docs/guides/endgame.md

```diff
@@ -108,30 +110,44 @@ endgame team is responsible for the following high-level milestones:
 
 ### Week 1: Project kickoff
 
+_Note: Week 1 and week 6 overlap. Previous release leader will only take charge
```

Comment:
I think this was just the case this past time, but ideally they shouldn't overlap, right?

Reply:
Ah, maybe @joshblack I assumed that was done on purpose since we began project planning (not sure who scheduled that) and stuff week 5. It can for sure be separate.